code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -366,7 +369,7 @@
 	..()
 	H << "<span class='warning'><b>You crumple in agony as your flesh wildly morphs into new forms!</b></span>"
 	H.visible_message("<b>[H]</b> falls to the ground and screams as their skin bubbles and froths!") //'froths' sounds painful when used with SKIN.
-	H.Weaken(3, 0)
+	H.Weaken(3, 0, 0)
```

Comment:
I changed the argument on the weaken() and adjustweakened() proc, their argument are now identical to the other status adjustment procs, so you only need Weaken(3,0) and AdjustWeakened(-2, 0) above.